platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -161,14 +161,18 @@ define(
          * @returns {boolean} falsy when a duplicate datum is given
          */
         ImageryController.prototype.updateHistory = function (datum) {
-            if (this.$scope.imageHistory.length === 0 ||
-                !_.isEqual(this.$scope.imageHistory.slice(-1)[0], datum)) {
-                var index = _.sortedIndex(this.$scope.imageHistory, datum, this.timeFormat.format.bind(this.timeFormat));
-                this.$scope.imageHistory.splice(index, 0, datum);
-                return true;
+            if (this.$scope.imageHistory.length !== 0) {
+                var datumTime       = this.timeFormat.format(datum);
```

Comment:
Minor style point. Aligning the RHS of variable assignments is not a style we use. It's not wrong at all, we just don't use it. Right now this code block is a little incongruous with the rest of the code, so in the interests of consistency let's remove the extra spaces.